rewrite-maven/src/main/java/org/openrewrite/maven/ChangeManagedDependencyGroupIdAndArtifactId.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven;
+
+import java.util.Optional;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.xml.ChangeTagValueVisitor;
+import org.openrewrite.xml.tree.Xml;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+@Value
+@EqualsAndHashCode(callSuper = true)
+public class ChangeManagedDependencyGroupIdAndArtifactId extends Recipe {

```

Comment:
Thinking we could just shorten this name to `ChangeManagedDependency`

Reply:
We could do that but I chose this name to be consistent with [ChangeDependencyGroupIdAndArtifactId](https://github.com/openrewrite/rewrite/blob/main/rewrite-maven/src/main/java/org/openrewrite/maven/ChangeDependencyGroupIdAndArtifactId.java).

Also, I could certainly envision adding `ChangeManagedDependencyScope` to be perfectly consistent with how standard dependencies are handled.

Let me know what you prefer, I will adjust.

Reply:
@tkvangorder btw, one option would be to rename [ChangeDependencyGroupIdAndArtifactId](https://github.com/openrewrite/rewrite/blob/main/rewrite-maven/src/main/java/org/openrewrite/maven/ChangeDependencyGroupIdAndArtifactId.java) and create a deprecated class that extends the new one with the old name. If you teach your documentation to not document deprecated classes, you end up with something pretty clean.

And it's not really a problem to keep the one with the old name around as you won't have to maintain it.